tests/rewards/test_reward_nets.py

```diff
@@ -272,6 +322,168 @@ def test_potential_net_2d_obs():
+
+
+@pytest.fixture
+def numpy_transitions() -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
+    """A batch of states, actions, next_states, and dones as np.ndarrays for Env2D."""
+    return (
+        np.zeros((10, 5, 5)),
+        np.zeros((10, 1), dtype=int),
+        np.zeros((10, 5, 5)),
+        np.zeros((10,)),
+    )
+
+
+def test_reward_ensemble_test_value_error(env_2d):
+    with pytest.raises(ValueError):
+        reward_nets.RewardEnsemble(
+            env_2d.action_space,
+            env_2d.observation_space,
+            num_members=0,
+        )
+
+
+def test_reward_ensemble_predict_reward_moments(
+    two_ensemble: reward_nets.RewardEnsemble,
+    numpy_transitions: tuple,
+):
+    # Test that the calculation of mean and variance is correct
+    two_ensemble.members[0].value = 0
+    two_ensemble.members[1].value = 0
+    mean, var = two_ensemble.predict_reward_moments(*numpy_transitions)
+    assert np.isclose(mean, 0).all()
+    assert np.isclose(var, 0).all()
+    two_ensemble.members[0].value = 3
+    two_ensemble.members[1].value = -1
+    mean, var = two_ensemble.predict_reward_moments(*numpy_transitions)
+    assert np.isclose(mean, 1).all()
+    assert np.isclose(var, 8).all()  # note we are using the unbiased variance estimator
+    # Test that ensemble calls members correctly
+    two_ensemble.members[0].forward = mock.MagicMock(return_value=th.zeros(10))
+    mean, var = two_ensemble.predict_reward_moments(*numpy_transitions)
+    two_ensemble.members[0].forward.assert_called_once()
+
+
+def test_ensemble_members_have_different_parameters(env_2d):
+    ensemble = reward_nets.RewardEnsemble(
+        env_2d.observation_space,
+        env_2d.action_space,
+        num_members=2,
+        member_cls=reward_nets.BasicRewardNet,
+    )
+
+    assert not th.allclose(
+        next(ensemble.members[0].parameters()),
+        next(ensemble.members[1].parameters()),
+    )
+
+
+def test_add_std_reward_wrapper(
+    two_ensemble: reward_nets.RewardEnsemble,
+    numpy_transitions: tuple,
+):
+    two_ensemble.members[0].value = 3
+    two_ensemble.members[1].value = -1
+    reward_fn = reward_nets.AddSTDRewardWrapper(two_ensemble, default_alpha=0.1)
+    rewards = reward_fn.predict_processed(*numpy_transitions)
+    assert np.allclose(rewards, 1 + 0.1 * np.sqrt(8))
+    # test overriding in predict processed works correctly
+    rewards = reward_fn.predict_processed(*numpy_transitions, alpha=-0.5)
+    assert np.allclose(rewards, 1 - 0.5 * np.sqrt(8))
+
+
+@pytest.mark.parametrize("make_wrapper", MAKE_BASIC_REWARD_NET_WRAPPERS)
+def test_wrappers_pass_on_kwargs(
+    make_wrapper: reward_nets.RewardNetWrapper,
+    env_2d: Env2D,
+    numpy_transitions: tuple,
+):
+    basic_reward_net = reward_nets.BasicRewardNet(
+        env_2d.observation_space,
+        env_2d.action_space,
+    )
+    basic_reward_net.predict_processed = mock.Mock(return_value=np.zeros((10,)))
+    normalized_reward_net = make_wrapper(
```

Comment:
This isn't necessarily normalized (can also be shaped from `MAKE_BASIC_REWARD_NET_WRAPPERS`)